scripts/globals/status.lua

```diff
@@ -1818,6 +1818,8 @@ xi.mod =
     BLUE_MAGIC_EFFECT       = 1059, -- TODO: Bonus to Attribute Value of spell (percent)
     QUICK_DRAW_RECAST       = 1060, -- TODO: Quick Draw Charge Reduction (seconds)
 
+    BREATH_DMG_DEALT        = 3033, -- Breath damage dealt
```

Comment:
you have to change the modifier ID here aswell
They must be the same in both modifier.h and status.lua